index.js

```diff
@@ -102,3 +100,50 @@ module.exports.isMACValid = function(mac) {
 
     return !(mac.length != 2 * MAC_BYTES || mac.match(/[^a-fA-F0-9]/));
 }
+
+module.exports.relay = function(mac, params) {
+
+        getIP(params).then(function(ip) {
+            
+            var magicPacket = createMagicPacket(mac);
+            listensocket = dgram.createSocket('udp4');
+            
+            listensocket.on('error', (err) => {
+              console.log(`server error:\n${err.stack}`);
+              listensocket.close();
+            });
+            
+            listensocket.on('message', (msg, rinfo) => {
+            
+              if (msg.equals(magicPacket))
```

Comment:
i don't have any codestyle linters, but i prefer this style
```js
if () {
}
```

Reply:
Sure, I'll change to be consistent with the rest of the code.